common/djangoapps/enrollment/urls.py

```diff
@@ -28,3 +33,8 @@
         name='courseenrollmentdetails'
     ),
 )
+
+if settings.FEATURES.get('ENABLE_BULK_ENROLLMENT_VIEW'):
```

Comment:
this should be plumbed-through the configuration helper in order to provide multi-site support